.github/workflows/run-tests.yml

```diff
@@ -60,10 +60,3 @@ jobs:
             ${{ runner.os }}-go-
       - name: Run linter and tests
         run: make test-coverage-custom
-      - name: Update code coverage
-        uses: codecov/codecov-action@v4.0.1
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          flags: unittests
-          fail_ci_if_error: true # optional (default = false)
-          verbose: true # optional (default = false)
```

Comment:
Just in case someone is wondering why it is deleted:
It is because codecov is failing with message about lack of rights to publish the raport.
We have a task [BUX-617](https://bitcoinassociation.atlassian.net/browse/BUX-617) to add code coverage reports again.